src/main/java/org/jabref/logic/importer/fileformat/PdfContentImporter.java

```diff
@@ -458,6 +465,9 @@ Optional<BibEntry> getEntryFromPDFContent(String firstpageContents, String lineS
             entry.setField(StandardField.PAGES, pages);
         }
         if (year != null) {
+            if (arxivId != null) {
+                year = "20" + arxivId.substring(0, 2);
+            }
```

Comment:
This should go to the `else` branch. - If there is no year then one can check for `arxivId` and guess a year out of it.